app/routes/__public/oauth/$id.tsx

```diff
@@ -163,6 +174,7 @@ const loadData = async ({
           process.env.NODE_ENV === "development" &&
           response.body.suggestExtension,
         postInstallResult,
+        accessToken: response.body.accessToken,
```

Comment:
[blocking] could we move this to the top level? we reuse `body` elsewhere so we expect that shape to be consistent for now